OpenNefia.Content/CharaMake/CharaMakeData.cs

```diff
@@ -14,5 +14,31 @@ public CharaMakeData()
         {
             CharaData = new Dictionary<Type, Dictionary<string, object>>();
         }
+
+        public bool TryGetValues(string key, out IEnumerable<object> vals)
+        {
+            vals = CharaData.Values.Where(x => x.TryGetValue(key, out _)).Select(x => x[key]);
+            return vals.Any();
+        }
+
+        /// <summary>
+        /// Returns the first object with the correct type in the data
+        /// </summary>
+        public bool TryGetValue<T>(string key, out T val)
```

Comment:
Maybe `val` should be nullable if this is a `TryGet` method. Also annotate `val` with `[NotNullWhen(true)]` if so.

Reply:
done in 24f46e9